string label
string file_path
string frame
float32 r
float32 g
float32 b
float32 a
---
bool success
string message
